# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/wr/WebRenderLayerManager.h
# Commit: 36b0646e8649
# Full Hash: 36b0646e86498662e770756ad54b1fd648a82184
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2020-04-07 21:44:02
# Regressor Bug: 1620594
# File Overlap Count: 1
# Description:
#   Bug 1620594 - Part 7: Remove TabGroup and SystemGroup. r=nika,bas
#   
#   TabGroup never really made any difference in which thread something go
#   dispatched to. This was the intended use, but development of TabGroups
#   with abstract main threads never made it that far. The good thing is
# ==============================================================================

diff -r a5ad3b179c92 -r 36b0646e8649 gfx/layers/wr/WebRenderLayerManager.h
--- a/gfx/layers/wr/WebRenderLayerManager.h	Tue Apr 07 15:17:14 2020 +0000
+++ b/gfx/layers/wr/WebRenderLayerManager.h	Tue Apr 07 15:17:47 2020 +0000
@@ -35,10 +35,6 @@
 
 struct ActiveScrolledRoot;
 
-namespace dom {
-class TabGroup;
-}
-
 namespace layers {
 
 class CompositorBridgeChild;
@@ -176,8 +172,6 @@
   void WrUpdated();
   nsIWidget* GetWidget() { return mWidget; }
 
-  dom::TabGroup* GetTabGroup();
-
   uint32_t StartFrameTimeRecording(int32_t aBufferSize) override;
   void StopFrameTimeRecording(uint32_t aStartIndex,
                               nsTArray<float>& aFrameIntervals) override;